### DISCUSSION TITLE
How to generate certificates?

### DISCUSSION THREAD

**Discussion by vdrover** [jlesage/docker-handbrake#380] [Q&A]
I'd like to enable user auth for the web gui but i have issues with my reverse proxy when I add the SECURE_CONNECTION flag. 

I read some prior issues on where to place the certificates, but I'm not sure how to generate the certificates in the first place. Note that a `cert` folder was not automatically generated in my app folder.

**Comment by jlesage** [comment]
The container will generate self-signed certificates by default if no one is provided. These are enough when the container used with a reverse proxy. The reverse proxy generates and manages certificates.
